pgjdbc/src/main/java/org/postgresql/PGArraySupport.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2004, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql;
+
+import java.sql.Array;
+import java.sql.SQLException;
+
+/**
+ * Defines public PostgreSQL extensions to create {@link Array} objects.
+ */
+public interface PGArraySupport {
```

Comment:
How much this level of indirection is useful? How many methods can be here potentially? I would suggest declaring the method directly in PGConnection.